rust-connectors/sources/postgres/postgres.conf

```diff
@@ -0,0 +1,798 @@
+# -----------------------------
```

Comment:
Is this whole file needed? It seems like a lot.

Reply:
The short answer is no, there are only 4 lines used.

The longer answer is that the connector does not actually use this file, but rather it is part of the Postgres database setup required before the connector can interact with it. This is the baseline example config that Postgres gives us, I think it stubs out all the possible or common settings and provides comments about what they are.

I would like to keep at least the relevant parts of this file (the 4 lines, and perhaps their comments) in here, but if you'd prefer for me to cut it down I can do that.

Reply:
Sorry, I wrote an answer to this days ago but I guess never pushed the submit button.

> I would like to keep at least the relevant parts of this file (the 4 lines, and perhaps their comments) in here, but if you'd prefer for me to cut it down I can do that.

Lets do this. Also, will these options be useful for writing an integration test? We should take advantage of that if we can.